### PR TITLE
Edit messages for various dev-related generate commands

### PR DESCRIPTION
**Pull request by jshackell-sfdc**
### What does this PR do?

Edits messages (help and errors) for various "sf generate " commands 

### What issues does this PR fix or reference?
@W-11716092@

### CODE REVIEW

File: messages/plugin.generator.md [hunk 1]
```diff
@@ -4,32 +4,32 @@ Time to build an sf plugin! Version %s
 
 # question.internal
 
-Are you building a plugin for an internal Salesforce team
+Are you building a plugin for an internal Salesforce team?
```
Comment (mdonnalley) [inline messages/plugin.generator.md:7]: The inquirer library puts `?` at the beginning of the question like this:

```
? Are you building a plugin for an internal Salesforce team (Y/n)
```

Do you think it'll look odd to have a ? on both ends?

```
? Are you building a plugin for an internal Salesforce team? (Y/n)
```

File: messages/dev.generate.library.md [hunk 2]
```diff
@@ -4,7 +4,9 @@ Generate a new library.
 
 # description
 
-This will clone the template repo 'forcedotcom/library-template' and update package properties
+This command is interactive. You're prompted for information to populate the new library, such as the npm scope (which must start with "@"), the name and description of the library, and its GitHub organization. The command clones the 'forcedotcom/library-template' GitHub repository, installs the plug-in's npm package dependencies using yarn install, and updates the package properties.
```
Comment (mdonnalley) [inline messages/dev.generate.library.md:7]: `installs plugin-in's npm...` -> `install library's npm ...`

**Review by mdonnalley — CHANGES_REQUESTED** [review]

File: messages/plugin.generator.md (hunk 1 above)
Comment (jshackell-sfdc) [reply messages/plugin.generator.md:7]: Good point.  But I think it's better to keep the two question marks.   The first one is a different color (in my terminal at least) and acts as a "this line is some kind of prompt" identifier, as opposed to the ">" character, for example, which identifies an option.  The verbiage itself is in bold, and without the ending question mark it looks incomplete and a mistake, IMHO.

We also followed this format (ending question mark) w/Shane's commands to generate objects/fields/etc, so it's nice to be consistent. 

So I'd prefer to keep the ending question mark.

**Review by mdonnalley — APPROVED** [review]

File: messages/dev.generate.library.md (hunk 2 above)
Comment (jshackell-sfdc) [reply messages/dev.generate.library.md]: Updated. Thx.

**Pull request merged**